### PR TITLE
Added harness mode for Next.js.

### PR DESCRIPTION
**Pull request by apascal07**

### CODE REVIEW

File: genkit-tools/common/src/runner/harness.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { glob } from 'glob';
+import * as path from 'path';
+import { logger } from '../utils';
+
+async function main() {
```
Comment (mbleigh) [inline genkit-tools/common/src/runner/harness.ts:5]: Will this work with loading/transpiling TS files? Does it automatically set `GENKIT_ENV=dev`?

**Review by mbleigh — COMMENTED** [review]
So the usage for this is:

```
genkit start src/ai/*.ts --mode harness
```

is that right?

**Comment by apascal07** [comment]
> So the usage for this is:
> 
> ```
> genkit start src/ai/*.ts --mode harness
> ```
> 
> is that right?

Not quite.

You add this to your genkit-tools.conf.ts:

```
module.exports = {
  runner: {
    mode: 'harness',
    files: ['./src/*.ts'],
  }
};```

And then run normally with `genkit start`.

File: genkit-tools/common/src/runner/harness.ts (hunk 1 above)
Comment (apascal07) [reply genkit-tools/common/src/runner/harness.ts:5]: Tested it with .ts, it worked. The envs are set automatically like in the default app loading.

**Review by mbleigh — APPROVED** [review]
Love it! Can you adjust `genkit init` for next.js to:

1. Create `genkit.config.ts` inside a `src/ai` directory
2. Automatically configure `genkit start` for harness mode

**Comment by apascal07** [comment]
> Love it! Can you adjust `genkit init` for next.js to:
> 
> 1. Create `genkit.config.ts` inside a `src/ai` directory
> 2. Automatically configure `genkit start` for harness mode

Yep, will do in follow up PR.

**Pull request merged**